ecommerce_api_client/auth.py

```diff
@@ -1,24 +1,40 @@
+# pylint: disable=duplicate-code
+
 import jwt
+import datetime
+
 from requests.auth import AuthBase
 
 
 class JwtAuth(AuthBase):
     """Attaches JWT Authentication to the given Request object."""
 
-    def __init__(self, username, full_name, email, signing_key, tracking_context=None):
+    def __init__(self, username, full_name, email, signing_key, tracking_context=None, issuer=None, expires_in=30):
+        self.issuer = issuer
+        self.expires_in = expires_in
         self.username = username
         self.email = email
         self.full_name = full_name
         self.signing_key = signing_key
         self.tracking_context = tracking_context
 
+    def datetime_now(self):
+        return datetime.datetime.utcnow()
+
     def __call__(self, r):
+
         data = {
             'username': self.username,
             'full_name': self.full_name,
-            'email': self.email,
+            'email': self.email
```

Comment:
@Nickersoft FYI: a valid reason to include the comma after the last item in a multiline parameter sequence is because it makes for a cleaner diff next time someone adds / removes a parameter.  That said, it's not something we do everywhere consistently, so reverting this is optional.